sensiolabs-de/storyblok-bundle/0.2/config/routes/storyblok.yaml

```diff
@@ -0,0 +1,2 @@
+#storyblok:
+#    resource: '@StoryblokBundle/config/routes.php'
```

Comment:
I would not comment it, the route doesn't hurt, if you a) don't add the route at Storyblok or b) don't create a handler for and event